File: rest-api-games/src/main/java/com/mycompany/rest/api/games/modelos/juegos/BusquedaJuego.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.rest.api.games.modelos.juegos;

/**
 *
 * @author edu
 */
public class BusquedaJuego {
    private String nombre;
    private double precioMaximo;
    private double precioMinimo;
    private String categoria;
    private String correoAdmin;

    public String getNombre() {
        return nombre;
    }

    public double getPrecioMaximo() {
        return precioMaximo;
    }

    public double getPrecioMinimo() {
        return precioMinimo;
    }

    public String getCategoria() {
        return categoria;
    }

    public String getCorreoAdmin() {
        return correoAdmin;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setPrecioMaximo(double precioMaximo) {
        this.precioMaximo = precioMaximo;
    }

    public void setPrecioMinimo(double precioMinimo) {
        this.precioMinimo = precioMinimo;
    }

    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }

    public void setCorreoAdmin(String correoAdmin) {
        this.correoAdmin = correoAdmin;
    }
    
    
    
    
    
    
}

File: games-app/src/models/juegos/busqueda-juego.ts
export class BusquedaJuego {
    nombre: string | null;
    precioMaximo: number;
    precioMinimo: number;
    categoria: string | null;
    correoAdmin: string | null;

    constructor(nombre: string| null, precioMax: number, precioMin:number, categoria:string | null, correo: string | null) {
        this.nombre = nombre;
        this.precioMaximo = precioMax;
        this.precioMinimo = precioMin;
        this.categoria = categoria;
        this.correoAdmin = correo;
    }
}